mobile/src/main/res/layout/openhabwidgetlist_rollershutteritem.xml

```diff
@@ -23,41 +23,44 @@
     <ImageButton
         android:id="@+id/rollershutterbutton_up"
         android:layout_width="46dip"
-        android:layout_height="46dip"
+        android:layout_height="0dp"
```

Comment:
Are those height changes intended? Keeping the explicit height of 46dp would probably make the sizing intent a bit clearer, even though the end result should be the same.

Reply:
Having the height set to match_constraint (0dp) makes them a bit larger (left one 46, right on 0 dp height):
![buttons](https://user-images.githubusercontent.com/22525368/36972299-eb45fdb2-206e-11e8-9e74-11b19ec82ba6.JPG)